### PR TITLE
updated readme files

### LINKED ISSUE #27: Change the README file to the English version
Change the README file to the English version as the main one, add in both README a link to their respective translations.

### PR DESCRIPTION
**Pull request by satyamrs00**
this closes #27

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -20,90 +20,92 @@
   </a>
 </p>
 
+[leer en español](readme_es.md)
```
Comment (Franqsanz) [inline readme.md:23]: Don't you think it would look better this way: 

example: 
[Leer en Español]()

Similarly for the English version.

Comment (Franqsanz) [reply readme.md:23]: You could also add the link like this

Examples:

```html 
<center><em><a href="readme_es.md">Leer en Español</a></em></center>
```
```html 
<center><em><a href="readme.md">Read this in English</a></em></center>
```

Comment (satyamrs00) [reply readme.md:23]: ok i will make these changes

**Comment by satyamrs00** [comment]
@Franqsanz I have made the changes now

**Comment by Franqsanz** [comment]
Good job, thanks for your contribution!

**Pull request merged**